src/main/java/ca/on/oicr/pineryreports/reports/impl/GazpachoProjectStatusReport.java

```diff
@@ -571,16 +684,27 @@ protected int getRowCount() {
     ReportItem info = sample.getValue();
     int i = -1;
     row[++i] = info.getStock().getName(); // stock name
+    row[++i] = ""; // placeholder for subproject, to be filled in by Jessica at this time
     row[++i] = info.getExternalName();
     row[++i] = info.getGroupIds();
     row[++i] = "1"; // received samples
     row[++i] = info.getWaitingAliquots(); // aliquots waiting
     row[++i] = info.getWaitingLibraries(); // libraries waiting
     row[++i] = info.getLibraries().isEmpty() ? "" : Integer.toString(info.getLibraries().size());// libraries created
     row[++i] = info.getWaitingSequencing(); // sequencing waiting
-    row[++i] = info.getNumLanesRunning(); // lanes running
-    row[++i] = info.getNumLanesCompleted(); // lanes completed
+    row[++i] = getNumLanes(info, RUN_RUNNING, MISEQ); // MiSeq lanes running
+    row[++i] = getNumLanes(info, RUN_COMPLETED, MISEQ); // MiSeq lanes completed
+    row[++i] = getNumLanes(info, RUN_RUNNING, HISEQ); // HiSeq lanes running
+    row[++i] = getNumLanes(info, RUN_COMPLETED, HISEQ); // HiSeq lanes completed
+    row[++i] = getNumLanes(info, RUN_RUNNING, NOVASEQ); // NovaSeq lanes running
+    row[++i] = getNumLanes(info, RUN_COMPLETED, NOVASEQ); // NovaSeq lanes completed
     row[++i] = info.getDaysInLab(end); // days in genomics
+    if (info.getStock().getName().equals("PCSI_0748_Lv_M_nn_5-1_D_S1")) {
+      System.out.println("Run count: " + info.getRuns().size());
+      for (ModifiedRunDto r : info.getRuns()) {
+        System.out.println("In the count: Run " + r.getRunId() + ", lanes " + r.getLanes().toString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)